### PR TITLE
UI development

### PR DESCRIPTION
**Pull request by imila254**
- Floating Menu button added.
- Transaction page design changed.
- Separate class for receipt and csv file upload created.
- Login page design changed.

image
image
image

### CODE REVIEW

File: MobileApplication/MobileApplication/App.xaml.cs [hunk 1]
```diff
@@ -14,6 +14,7 @@ public App()
             InitializeComponent();
 
             //DependencyService.Register<MockDataStore>();
+            Device.SetFlags(new[] { "Expander_Experimental" });
```
Comment (AleksandrasMaliuginas) [inline MobileApplication/MobileApplication/App.xaml.cs:17]: What does this Flag do?

File: MobileApplication/MobileApplication/ViewModels/AddFileAndImage.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
+﻿using MobileApplication.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using Xamarin.Essentials;
+
+namespace MobileApplication.ViewModels
+{
+    public  class AddFileAndImage
```
Comment (AleksandrasMaliuginas) [inline MobileApplication/MobileApplication/ViewModels/AddFileAndImage.cs:10]: Maybe the better name could be something like: `MediaUploader` or `TransactionUploader` with methods something like UploadReceipt( Image ) and UploadBankStatment( csvFile )

File: MobileApplication/MobileApplication/Views/Transaction/ItemsPage.xaml [hunk 3]
```diff
@@ -16,32 +16,122 @@
...
+            <Grid ColumnSpacing="0" HorizontalOptions="FillAndExpand">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="*"/>
+                </Grid.ColumnDefinitions>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="*"/>
+                    <RowDefinition Height="Auto"/>
+                </Grid.RowDefinitions>
+                <BoxView Grid.RowSpan="2" WidthRequest="80" BackgroundColor="#00c6ae"/>
```
Comment (AleksandrasMaliuginas) [inline MobileApplication/MobileApplication/Views/Transaction/ItemsPage.xaml:51]: This huge comment could be deleted, right?

**Review by AleksandrasMaliuginas — APPROVED** [review]
Marvellous improvements! I really like transactions page and the floating button is awesome ux feature!

File: MobileApplication/MobileApplication/Views/Transaction/ItemsPage.xaml (hunk 3 above)
Comment (imila254) [reply MobileApplication/MobileApplication/Views/Transaction/ItemsPage.xaml:51]: Yes, I just had a few more ideas that have failed so far. Will delete it.

File: MobileApplication/MobileApplication/ViewModels/AddFileAndImage.cs (hunk 2 above)
Comment (imila254) [reply MobileApplication/MobileApplication/ViewModels/AddFileAndImage.cs:10]: I will change it.

File: MobileApplication/MobileApplication/App.xaml.cs (hunk 1 above)
Comment (imila254) [reply MobileApplication/MobileApplication/App.xaml.cs:17]: This indicated that the function (Expander, used in floating menu button) is still in experimental state and awaiting feedback. Without it program throws exception.

**Review by barkas90 — APPROVED** [review]
Looks fantastic

**Pull request merged**